docs/getting-started.rst

```diff
@@ -98,69 +92,59 @@ Required columns in input data
 +--------------+---------------------------------------------------------+-------------+
-|renteneintritt|                                                         | Int         |
+|miete         |Monthly rent expenses (without heating)                  | Float       |
 +--------------+---------------------------------------------------------+-------------+
-|handcap_degree|                                                         | Int         |
+|heizkost      |Monthly heating expenses                                 | Float       |
 +--------------+---------------------------------------------------------+-------------+
-|wohnfl        |                                                         | Int         |
+|renteneintritt|retirement age                                           | Int         |
 +--------------+---------------------------------------------------------+-------------+
-|zveranl       |                                                         | Bool        |
+|handcap_degree|Handicap Degree (between 0 and 100                       | Int         |
 +--------------+---------------------------------------------------------+-------------+
-|ineducation   |                                                         | Bool        |
+|wohnfl        |size of dwelling in sqm                                  | Int         |
 +--------------+---------------------------------------------------------+-------------+
-|alleinerz     |                                                         | Bool        |
+|zveranl       |Dummy: married couple filing jointly for income tax      | Bool        |
 +--------------+---------------------------------------------------------+-------------+
-|eigentum      |                                                         | Bool        |
+|ineducation   |Dummy: employment status "in education"                  | Bool        |
 +--------------+---------------------------------------------------------+-------------+
-|child11_num_tu|                                                         | Int         |
+|alleinerz     |Dummy: Single Parent                                     | Bool        |
 +--------------+---------------------------------------------------------+-------------+
-|cnstyr        |                                                         | Int         |
+|eigentum      |Dummy: Owner-occupied housing                            | Bool        |
 +--------------+---------------------------------------------------------+-------------+
-|m_transfers   |                                                         | Int         |
+|cnstyr        |Constr. Yr of Dwelling (1: <1965,2:1966-2000,3:>2000)    | Int         |
 +--------------+---------------------------------------------------------+-------------+
-|hh_korr       |                                                         | Int         |
+|m_transfers   |Sum of monthly public/pricate transfers not simulated    | Int         |
```

Comment:
private

What is it supposed to mean? We can have more than one row, can't we?

Reply:
I guess this variable comes from SOEP where people are probably asked: Do you have other transfers?